recipe/migrations/cuda120.yaml

```diff
@@ -0,0 +1,72 @@
+migrator_ts: 1682985063
+__migrator:
+  kind:
+    version
+  use_local: true  # TODO: Remove this line after completing testing
```

Comment:
Adding to simplify testing. Though this needs to be dropped after. Flagging as a reminder

```suggestion
```